src/components/Calculator.js

```diff
@@ -1,86 +1,77 @@
-            <button className="colorW" type="button" name="AC" onClick={(e) => { this.getInf(this.state, e.target.name); }}>AC</button>
-            <button className="colorW" type="button" name="+/-" onClick={(e) => { this.getInf(this.state, e.target.name); }}>+/-</button>
-            <button className="colorW" type="button" name="%" onClick={(e) => { this.getInf(this.state, e.target.name); }}>%</button>
-            <button className="colorS" type="button" name="÷" onClick={(e) => { this.getInf(this.state, e.target.name); }}>÷</button>
-          </div>
-          <div className="conSecond">
-            <button className="colorW" type="button" name="7" onClick={(e) => { this.getInf(this.state, e.target.name); }}>7</button>
-            <button className="colorW" type="button" name="8" onClick={(e) => { this.getInf(this.state, e.target.name); }}>8</button>
-            <button className="colorW" type="button" name="9" onClick={(e) => { this.getInf(this.state, e.target.name); }}>9</button>
-            <button className="colorS" type="button" name="x" onClick={(e) => { this.getInf(this.state, e.target.name); }}>x</button>
-          </div>
-          <div className="conThird">
-            <button className="colorW" type="button" name="4" onClick={(e) => { this.getInf(this.state, e.target.name); }}>4</button>
-            <button className="colorW" type="button" name="5" onClick={(e) => { this.getInf(this.state, e.target.name); }}>5</button>
-            <button className="colorW" type="button" name="6" onClick={(e) => { this.getInf(this.state, e.target.name); }}>6</button>
-            <button className="colorS" type="button" name="-" onClick={(e) => { this.getInf(this.state, e.target.name); }}>-</button>
-          </div>
-          <div className="conFourth">
-            <button className="colorW" type="button" name="1" onClick={(e) => { this.getInf(this.state, e.target.name); }}>1</button>
-            <button className="colorW" type="button" name="2" onClick={(e) => { this.getInf(this.state, e.target.name); }}>2</button>
-            <button className="colorW" type="button" name="3" onClick={(e) => { this.getInf(this.state, e.target.name); }}>3</button>
-            <button className="colorS" type="button" name="+" onClick={(e) => { this.getInf(this.state, e.target.name); }}>+</button>
-          </div>
-          <div className="conFifth">
-            <button className="colorW" type="button" name="0" onClick={(e) => { this.getInf(this.state, e.target.name); }}>0</button>
-            <div>
-              <button className="colorW" type="button" name="." onClick={(e) => { this.getInf(this.state, e.target.name); }}>.</button>
-              <button className="colorS" type="button" name="=" onClick={(e) => { this.getInf(this.state, e.target.name); }}>=</button>
-            </div>
+  return (
+    <div className="calContainer">
+      <input className="calInput" placeholder="0" value={`${format()}`} disabled="disabled" />
```

Comment:
- [ ] It looks like there's a bug in the display calculator. When I try an operation like `5+6` the answer includes the sign for this at the end. I kindly invite you to work on this. 🐶 
![image](https://user-images.githubusercontent.com/20563252/129353459-a30fb8cb-e098-4b15-932e-56d94c4b156b.png)